### PR TITLE
fix: [CDS-53377]: added harness (local) store type under store types to show c…

### PR DESCRIPTION
**Pull request by achyuth-harness**
…ommand flags

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
    - CodeFormat: `trigger codeformat`
    - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- TI-ALL: `trigger tiAll`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/43989)

### CODE REVIEW

File: 127-cd-nextgen-entities/src/main/java/io/harness/cdng/manifest/ManifestStoreType.java [hunk 1]
```diff
@@ -59,8 +59,8 @@ static boolean isInStorageRepository(String manifestType) {
     }
   }
 
-  Set<String> HelmChartRepo = ImmutableSet.of(HTTP, GCS, S3, OCI, CUSTOM_REMOTE);
+  Set<String> HelmChartRepo = ImmutableSet.of(HTTP, GCS, S3, OCI, CUSTOM_REMOTE, HARNESS);
```
Comment (abosii-harness) [inline 127-cd-nextgen-entities/src/main/java/io/harness/cdng/manifest/ManifestStoreType.java:62]: We don't want to support for Harness store type pull, fetch, repo add etc

**Review by abosii-harness — CHANGES_REQUESTED** [review]

**Review by achyuth-harness — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43989)* status: 1 file(s) to review. 1 change requests. 1. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`127-cd-nextgen-entities/src/main/java/io/harness/cdng/manifest/ManifestStoreType.java` line 62 at r1](https://reviewable.io/reviews/harness/harness-core/43989#-NOJUTAnfnXj9ihyoRPS-r1-62:-NOK6JEP5Aaaw9-Ue2vR:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/0dcc42a8f39077d3f6b04ebe3a392976d8e985e5/127-cd-nextgen-entities/src/main/java/io/harness/cdng/manifest/ManifestStoreType.java#L62)):*
<details><summary><i>Previously, abosii-harness (Alexandru Bosii) wrote…</i></summary><blockquote>

We don't want to support for Harness store type pull, fetch, repo add etc
</blockquote></details>

Done.

**Review by GoelPratyush99 — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43989)* status: 1 file(s) to review. 1 change requests. 1. unresolved discussions. 1 of 2 approvals obtained.

**Review by abosii-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43989)* status: Some of the checks are too old.

**Comment by achyuth-harness** [comment]
trigger messagecheck

**Pull request merged**